service/src/main/java/com/ge/predix/acs/service/policy/admin/PolicyManagementServiceImpl.java

```diff
@@ -150,7 +149,7 @@ public void deletePolicySet(final String policySetID) {
                     policySetID, zone.getName()));
             // Since we only support one policy set and we don't want to load that policy set when checking for a
             // cached invalidation, we use a hard-coded value for the policy set key.
-            this.cache.resetForPolicySet(zone.getName(), "default");
+            this.cache.resetForPolicySet(zone.getName(), policySetID);
```

Comment:
Please consider strict camel casing "policySetID" -> "policySetId". Also consider that: 1) "Id" is an abbreviation for identifier and not an acronym so it doesn't even make sense to capitalize the "D" 2) everywhere else we use "Id" instead of "ID" let's be consistent.